app/assets/javascripts/module/tree.js.coffee

```diff
@@ -1,5 +1,8 @@
 class Module.Tree
   @load: (loading_html, module_id) ->
+    # FIXME: The messages we send on loading_html are the same already
```

Comment:
Not sure if this is a FIXME. I believe that this load message setter is there for the case where the user ask to reprocess or ask for a processing with specific date.

What do you think? Can we remove it?